src/main/java/gui/viewmodel/SettingsViewModel.java

```diff
@@ -0,0 +1,4 @@
+package gui.viewmodel;
+
+public class SettingsViewModel extends ViewModel{
+}
```

Comment:
Do we need this class since it has no content?

Reply:
xander will be using it for ics exports